src/components/elements/carousel/carousel-style.tsx

```diff
@@ -40,18 +52,18 @@ export const EmblaSlideInner = styled.div`
     }}
 `
 
-export const StyledButtonWrapper = styled.div`
+export const StyledButtonWrapper = styled.div<Props>`
     position: absolute;
     bottom: 50%;
-    opacity: ${(props) => (props.disabled ? '0.32' : '1')};
-    ${(props) => {
-        if (props.left)
+    opacity: ${(disabled) => (disabled ? '0.32' : '1')};
+    ${(left) => {
+        if (left)
```

Comment:
Have you check if `props.disabled` and `disabled` is giving thesame value?

Reply:
No, I hadn't checked it. But whether I put props.disabled or disabled I don't see any changes.

Reply:
If we use disabled as props, the opacity must be changed.
May you double check `disabled` and `left`?
Since we are passing `props` and it's an object, I don't think it would work now.